### PR TITLE
Update test to run all

### LINKED ISSUE #274: Test `deploy_service` and `node_pool` fail verification on CI
All tests passed locally. 
If I run only 2 tests (`deploy_service` and `node_pool`) on CI - verification passed.
Run all tests at once on CI, got error on step `verify` with `kubectl`: 
```
$$$$$$ Running command `terraform output -json` in directory /workspace/test/fixtures/deploy_service
deploy_service: Verifying

Profile: deploy_service
Version: (not specified)
Target: local://

✔ gcloud: Google Compute Engine GKE configuration
✔ Command: `gcloud --project=ci-gke-41bd container clusters --zone=us-east4 describe deploy-service-cluster-59wd --format=json` exit_status should eq 0
✔ Command: `gcloud --project=ci-gke-41bd container clusters --zone=us-east4 describe deploy-service-cluster-59wd --format=json` stderr should eq ""
✔ Command: `gcloud --project=ci-gke-41bd container clusters --zone=us-east4 describe deploy-service-cluster-59wd --format=json` cluster is running
× kubectl: Kubernetes configuration (3 failed)
× kubernetes services nginx exists
Unauthorized
× kubernetes services nginx has an IP address
Unauthorized
× kubernetes services nginx is reachable
expected no Exception, got #<Kubeclient::HttpError: HTTP status code 401, Unauthorized for GET https://35.245.158.112/api/v1> with backtrace:


Profile Summary: 1 successful control, 1 control failure, 0 controls skipped
Test Summary: 3 successful, 3 failures, 0 skipped
>>>>>> deploy_service: InSpec Runner exited with 1
-----> Setting up <disable-client-cert-local>...
Finished setting up <disable-client-cert-local> (0m0.00s).
-----> Verifying <disable-client-cert-local>...
$$$$$$ Running command `terraform workspace select kitchen-terraform-disable-client-cert-local` in directory /workspace/test/fixtures/disable_client_cert
$$$$$$ Running command `terraform output -json` in directory /workspace/test/fixtures/disable_client_cert
disable_client_cert: Verifying


-----> Setting up <node-pool-local>...
Finished setting up <node-pool-local> (0m0.00s).
-----> Verifying <node-pool-local>...
$$$$$$ Running command `terraform workspace select kitchen-terraform-node-pool-local` in directory /workspace/test/fixtures/node_pool
$$$$$$ Running command `terraform output -json` in directory /workspace/test/fixtures/node_pool
node_pool: Verifying

Profile: node_pool
Version: (not specified)
Target: local://

✔ gcloud: Google Compute Engine GKE configuration
✔ ...
× kubectl: Kubernetes configuration (2 failed)
× kubernetes nodes pool-01 has the expected taints
Unauthorized
× kubernetes nodes pool-02 has the expected taints
Unauthorized

```

### PR DESCRIPTION
**Pull request by paulpalamarchuk**
* Fixed #274

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -110,14 +104,14 @@ suites:
 #      systems:
 #        - name: stub_domains_private
 #          backend: local
-  - name: "upstream_nameservers"
+  - name: "upstream_nameservers" #
```
Comment (kopachevsky) [inline .kitchen.yml]: what this # for?

Comment (paulpalamarchuk) [reply .kitchen.yml]: Leftover removed

File: test/setup/main.tf [hunk 2]
```diff
@@ -14,15 +14,18 @@
  * limitations under the License.
  */
 
+resource "random_id" "random_project" {
```
Comment (aaron-lane) [inline test/setup/main.tf]: What is the motivation for this change? The removed `random_project_id` argument provides this behaviour.

Comment (paulpalamarchuk) [reply test/setup/main.tf]: While I was testing different branches at the same time locally, all the projects had the same name, so I've changed it.

Comment (aaron-lane) [reply test/setup/main.tf]: I'm not clear why that would be the case; `random_project_id = true` toggles this [same behaviour within Project Factory](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/core_project_factory/main.tf#L32-L36). Please revert this change.

Comment (paulpalamarchuk) [reply test/setup/main.tf]: reverted

**Comment by morgante** [comment]
@paulpalamarchuk Please make sure Cloud Build tests pass, you might need to rebase on master.

**Comment by morgante** [comment]
@aaron-lane PTAL
